src/codec/ValueCodec.ts

```diff
@@ -23,8 +23,8 @@ export const ValueCodec: Codec<PbValue, Value> = {
             return {valueType: 'contractId', contractId: value.getContractId()};
         } else if (value.hasDate()) {
             return {valueType: 'date', date: '' + value.getDate()};
-        } else if (value.hasDecimal()) {
-            return {valueType: 'decimal', decimal: value.getDecimal()};
+        } else if (value.hasNumeric()) {
```

Comment:
Is this right? Value can be a numeric that is not a decimal.

Reply:
Are numeric part of 1.6 already?

Reply:
No but they are part of 1.7 which is part of 0.13.55. If I use the bindings to listen on a transaction stream, I should be able to receive a numeric. What will happen in that case?

Reply:
Ok, sorry for the confusion here. This is an API Numeric, which is going to be always be a string. So this should not really be a problem. Makes sense?

Reply:
kinda, you are still setting `valueType` to `decimal` which is wrong if you don’t have a Numeric 10. But happy to keep it for now. Maybe just add a comment.

Reply:
I would prefer existing code to not break regarding this. If we want, we can change it later.